testcases/smbtorture-test/smbtorture-tests-info.yml

```diff
@@ -0,0 +1,9 @@
+---
+- smb2.rw:
+    expected_ret: True
+
+- smb2.read:
+    expected_ret: True
+
+- smb2.dir:
```

Comment:
As discussed, remove the dir test for now. The read (and hopefully the rw test?) should work once #68 is merged.